zkevm-circuits/src/tx_circuit.rs

```diff
@@ -475,15 +467,18 @@ mod tx_circuit_tests {
         const MAX_TXS: usize = 2;
         const MAX_CALLDATA: usize = 32;
 
-        let mut rng = ChaCha20Rng::seed_from_u64(2);
-        let chain_id: u64 = 1337;
-        let mut txs = Vec::new();
-        for _ in 0..NUM_TXS {
-            txs.push(rand_tx(&mut rng, chain_id));
-        }
-
         let k = 19;
-        assert_eq!(run::<Fr, MAX_TXS, MAX_CALLDATA>(k, txs, chain_id), Ok(()));
+        assert_eq!(
+            run::<Fr, MAX_TXS, MAX_CALLDATA>(
+                k,
+                mock::CORRECT_MOCK_TXS[..NUM_TXS]
```

Comment:
These txs are no longer generated from an rng, so I think changing the test name is necessary.
I suggest calling this test `serial_test_tx_circuit_2txs`, and the next one `serial_test_tx_circuit_1tx`.

Reply:
Well, the `from` is random. Same as it was before. Hence why I kept the test names. But happy to change them if you still consider it! :)

Reply:
Ah I see... But also now in the `serial_test_tx_circuit_fixed` the `from` is random.  My point is that now `serial_test_tx_circuit_rng` and `serial_test_tx_circuit_fixed` use the same vectors, and the only difference is that one uses 1 tx and the other one uses 2 txs.  So I think updating the test names will avoid confusion.  The new names were a suggestion, if you have another name idea that's also good!

Reply:
Fixed in b25452b